packages/react-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bucketco/react-sdk",
-  "version": "2.5.2",
+  "version": "2.6.0",
```

Comment:
this will immediately release a new minor version. Let's get this in first and get it tested well and then do a release.